agent/acs/handler/acs_handler_test.go

```diff
@@ -777,7 +777,7 @@ func TestConnectionIsClosedOnIdle(t *testing.T) {
 	<-connectionClosed
 }
 
-func TestHandlerDoesntLeakGouroutines(t *testing.T) {
+func TestHandlerDoesntLeakGoRoutines(t *testing.T) {
```

Comment:
"goroutine" is the noun here; I'd change this back to `TestHandlerDoesntLeakGoroutines`.